boards/arduino_nano33iot/README.md

```diff
@@ -1,32 +1,42 @@
-jkkkkk Arduino Nano 33 IOT Board Support Crate
+# Arduino Nano 33 IOT Board Support Crate
 
 This crate provides a type-safe API for working with the [Arduino nano 33 IOT board](https://store.arduino.cc/usa/nano-33-iot).
 
 ## Examples
-### Blinky Basic
-#### Requirements
- - Arduino IDE installed
-    - samd package installed (You can do this by going to Tools->Board->BoardManager and then searching for `samd`
-    - Now the arduino distribution contains bossac.exe in `ArduinoData/packages/arduino/tools/bossac/1.7.0[-arduino3]/` add it to your path
-       - **linux**: `ArduinoData` is likely something like `~/.arduino15/`
-       - **OSX**: `ArduinoData` is likely something like `~/Library/Arduino15`
-    - Probably best to install an example sketch via the IDE just to make sure everything is working
-    - Note that the [arduino cli](https://github.com/arduino/arduino-cli) (or just regular bossac) may soon replace this section
- - arm-none-eabi tools installed, you need gcc and objcopy.
-   -  **Note**: Alternatively, you can use [cargo-binutils](https://github.com/rust-embedded/cargo-binutils), which is likely easier to install on OSX and also easier to use, as it will automatically detect the target
- - thumbv6m-none-eabi rust target installed via `rustup target add thumbv6m-none-eabi`
-
-#### Steps
+
+* Blinky basic
```

Comment:
I don't like having the examples listed in the README. Too burdensome to update if new ones get added.

Reply:
One was already listed so just tidied it up, but I agree, it's easy enough to find them so no need to list